src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -172,6 +182,16 @@ public int getInsertThrottleThresholdInPercentage() {
     }
     return (int) val;
   }
+  /** @return Absolute size in bytes of free total memory at which we throttle row inserts */
+  public int getInsertThrottleThresholdInBytes() {
+    Object val =
+        this.parameterMap.getOrDefault(
+            INSERT_THROTTLE_THRESHOLD_IN_BYTES, INSERT_THROTTLE_THRESHOLD_IN_BYTES_DEFAULT);
+    if (val instanceof String) {
+      return Integer.parseInt(val.toString());
```

Comment:
Any chance to add a test for this?

Reply:
I extended the `ParameterProviderTest`. 
> Consider uploading reports for the commit https://github.com/snowflakedb/snowflake-ingest-java/commit/9f186da9e3036d941bf28abb53c785d8e8a03051 to get more accurate results

Any idea how can I make the CodeCov to give a new report?